src/libraries/System.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/JSHost.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Threading.Tasks;
+using System.Runtime.Versioning;
+using System.Threading;
+
+namespace System.Runtime.InteropServices.JavaScript
+{
+    /// <summary>
+    /// Represent JavaScript host environment like browser, in which the dotnet runtime is embedded.
+    /// </summary>
+    [SupportedOSPlatform("browser")]
+    public static partial class JSHost
+    {
+        /// <summary>
+        /// Returns proxy of the <see href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/globalThis">globalThis</see>, the top level object.
+        /// </summary>
+        public static JSObject GlobalThis
+        {
+            get
+            {
+                throw new NotImplementedException();
+            }
+        }
+
+        /// <summary>
+        /// Returns proxy of the runtime module export in JavaScript.
+        /// </summary>
+        public static JSObject DotnetInstance
+        {
+            get
+            {
+                throw new NotImplementedException();
+            }
+        }
+
+        /// <summary>
+        /// Download and instantiate ES6 module from provided URL.
+        /// It will use <see href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import">dynamic import</see> of underlying JavaScript engine.
+        /// The instance of the module would be downloaded only once per <paramref name="moduleName"/> and cached.
+        /// </summary>
+        /// <param name="moduleName">Globally unique identifier of the ES6 module, which is used by <see cref="JSImportAttribute(string, string)"/>.</param>
+        /// <param name="moduleUrl"></param>
+        /// <param name="cancellationToken">A cancellation token to observe.</param>
+        /// <returns>Proxy object of the module exports.</returns>
```

Comment:
```suggestion
        /// Download and instantiate an ES6 module from the provided URL, via the JavaScript host's <see href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import">dynamic import API</see>.
        /// If a module with the provided <paramref name="moduleName"/> has previously been instantiated, it will be returned instead.
        /// </summary>
        /// <param name="moduleName">Globally unique identifier of the ES6 module, which is used by <see cref="JSImportAttribute(string, string)"/>.</param>
        /// <param name="moduleUrl">The location of the module file.</param>
        /// <param name="cancellationToken">The token to monitor for cancellation requests.</param>
        /// <returns>A proxy for the JavaScript object that contains the module's exports.</returns>
```